src/components/forums/Forum.js

```diff
@@ -262,8 +358,50 @@ const Forum = () => {
                     </Card>
                 ))}
             </Box>
+            {selectedComment && (
+                <div>
+                    {/* Edit comment diaglog */}
+                    < Dialog open={isDialogOpen} onClose={handleCloseDialog} >
+                        <DialogTitle>Edit Your Comment</DialogTitle>
+                        <DialogContent>
+                            <Comment
+                                comment={comment}
```

Comment:
The edit Comment textfield is a little small. For example, if I want to change a comment to be a longer sentence, I can't see the majority of it as I type.

<img width="451" alt="image" src="https://user-images.githubusercontent.com/90345332/225354307-c7ec8f08-7523-4fd3-97a1-d25890f05285.png">

Reply:
<img width="455" alt="Screenshot 2023-03-15 at 12 12 57 PM" src="https://user-images.githubusercontent.com/67078410/225372020-55cac241-8883-4ff5-aae7-fdd4ad108016.png">

It is full-width, so I am not sure... will check again though! thank you